crates/blenvy/Cargo.toml

```
@@ -14,12 +14,12 @@ license = "MIT OR Apache-2.0"
 workspace = true
 
 [dependencies]
-bevy = { version = "0.14", default-features = false, features = ["bevy_asset", "bevy_scene", "bevy_gltf", "animation"] }
+bevy = { version = "0.15.0-rc.1", default-features = false, features = ["bevy_asset", "bevy_scene", "bevy_gltf", "animation"] }
```

```suggestion
bevy = { version = "0.15.0", default-features = false, features = ["bevy_asset", "bevy_scene", "bevy_gltf", "animation"] }
```

bevy `0.15` has been released now and thus should be updated here.